### DISCUSSION TITLE
Table 4 Members

### DISCUSSION THREAD

**Discussion by EleonoraMilea** [IBAHCM/RPiR#416] [Old]
Eleonora Milea: EleonoraMilea: https://github.com/IBAHCM/EleonoraMilea_series01.git

**Comment by JumaMinya** [comment]
Juma Minya: JumaMinya https://github.com/IBAHCM/JumaMinya_0201.git

**Reply by EleonoraMilea** [reply]
Hi :) it is not working. That's the error message I am getting: 

remote: Repository not found.
fatal: repository 'https://github.com/IBAHCM/JumaMinya_0201.git/' not found

**Reply by JumaMinya** [reply]
Even when connecting yours doesnt work with similar error
remote: Repository not found.
fatal: repository 'https://github.com/IBAHCM/EleonoraMilea_series01.git/' not found

**Reply by JumaMinya** [reply]
Hi,
I have just uploaded the practical 2-2
Kindly check and advice if those are the graphs

**Reply by EleonoraMilea** [reply]
Hey Juma, I can't see your second repository. The one above is the one from week 1 with the births-deaths model

**Reply by EleonoraMilea** [reply]
I can't even find it where all the other repositories are... not sure why, sorry!

**Reply by JumaMinya** [reply]
This has been sorted

**Comment by EleonoraMilea** [comment]
Eleonora Milea: EleonoraMilea: https://github.com/IBAHCM/EleonoraMilea_series02.git 

Hi @JumaMinya  I would appreciate if you could check my code up to 2.3. Thank you :)

**Reply by JumaMinya** [reply]
@EleonoraMilea 
Gone through the codes, seems ok.

**Reply by EleonoraMilea** [reply]
Thanks :)

**Reply by JumaMinya** [reply]
Hi @EleonoraMilea 
How long does it take for the github to update the push?
I have commit and push the practical 2-4 but that is not reflected (updated in github)

**Reply by EleonoraMilea** [reply]
Hi :) it generally takes few minutes when I do it. 
I have now updated the whole practical 2. Would you please run all codes in your computer and let me know if it all works fine? thank you :)

**Reply by EleonoraMilea** [reply]
oh, and raise an issue in my repository as well, thank you!
